Makefile.e2e

```diff
@@ -21,7 +21,6 @@ e2e-staging: \
 	@rm -rf $(E2E_DIR)
 	@mkdir -p $(E2E_DIR)
 	@mkdir -p $(E2E_DIR)/third_party
```

Comment:
Do we still need the `e2e-staging` target?

Reply:
You can also check other targets that depend on `e2e-staging` and confirm if they're still needed.

Reply:
it looks like `e2e-staging` is currently used for `make manual-test-boostrap` and `make manual-test-clean`. I'm not very familiar with the usage of these 2 make target, should they be removed or modified?

Reply:
I would suggest looking into these two manual-test-xxx targets and figuring out whether they're needed.

Reply:
i did some searching and looks like they are intended for local nomos testing installation. Since we moved to OSS, I think they are no longer needed and can be removed. http://b/158858700

Reply:
I don't understand why the local nomos tesitng installation is not needed after we moved to OSS.

But these two targets use the bats script to install Config Sync, that's why they can be removed.
You can find the details by looking at the `__test-e2e-run` target.

Reply:
+1 these look like convenience targets for developers to interact with the bats tests. They don't seem to be used by anything else, in which case they can be removed

Reply:
oh sorry, I mean because we are now maintaining another way to build and run from source as listed here https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/docs/installation.md#building-and-installing-from-source. Thanks for the clarification!